### PR TITLE
Always search for preludes if is not x86

### LINKED ISSUE #995: Some bug with `aaef` by `aaa`
| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits                             | Debian x86_64
| File format of the file you reverse       | ELF64
| Architecture/bits of the file          | ARM aarch64
| `rizin -v` full output, **not truncated**  | rizin 0.3.0-git @ linux-x86-64 commit: `63fbfd1cafa331c2024f05f2fd3936143b82dde1`, build: 2021-04-11__15:00:31

### Expected behavior
- `aaaa` should do `aaef` judging by output
### Actual behavior
- after `aaaa` at least one xref don't found
- but after separate `aaef`, it works fine
### Steps to reproduce the behavior
- Open this [binary](https://github.com/rizinorg/rizin/files/6291929/binary.zip) in rizin
- use `aaaa` and `axt @ 0x004bd638` - it wouldn't show xref
- use `aaef` and `axt @ 0x004bd638` - it will show xref

### Additional Logs, screenshots, source code,  configuration dump, ...
Screenshot with bug:
изображение

[binary](https://github.com/rizinorg/rizin/files/6291929/binary.zip)

### PR DESCRIPTION
**Pull request by wargio**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Fix #995

### CODE REVIEW

**Review by XVilka — APPROVED** [review]
Nice, new results are better

**Comment by XVilka** [comment]
Now it timeouts more often on:
```
[XX] TIMEOUT db/archos/linux-x64/dbg_aaa_dcu dbg.aaa
RZ_NOPLUGINS=1 /home/runner/bin/rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -eflirt.sigdb.load.system=false -eflirt.sigdb.load.home=false -N -dA -Qc 'dcu main
fd
afi @ main~?name: main
' bins/elf/true
-- stdout
--- expected
+++ actual
@@ -1,2 +1,0 @@
-main
-1

-- stderr
Process with PID 22751 started...
[ ] Analyze all flags starting with sym. and entry0 (aa)
[
[x] Analyze all flags starting with sym. and entry0 (aa)

[ ] Analyze function calls
[
[x] Analyze function calls
[ ] find and analyze function preludes
[
-- exit status: -1
```

**Comment by XVilka** [comment]
It is so much slower now on Windows

File: test/db/archos/linux-x64/dbg_aaa_dcu [hunk 1]
```diff
@@ -1,7 +1,8 @@
 NAME=dbg.aaa
```
Comment (XVilka) [inline test/db/archos/linux-x64/dbg_aaa_dcu]: Then name should be changed too.

**Review by XVilka — CHANGES_REQUESTED** [review]
Seems it affects the performance too much. Likely there is a huge bottleneck somewhere in the analysis. Have you tried to check the flamegraph for this timeouted test?

**Comment by wargio** [comment]
so, we don't really need to search for preludes on x86.

**Review by XVilka — APPROVED** [review]

**Comment by XVilka** [comment]
@wargio it broke fuzzing tests:
```
[XX] TIMEOUT /home/runner/work/rizin/rizin/test/bins/fuzzed <fuzz> /home/runner/work/rizin/rizin/test/bins/fuzzed/poc-radare2-invalid-memory-access-elf-758
RZ_NOPLUGINS=1 /home/runner/bin/rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -eflirt.sigdb.load.system=false -eflirt.sigdb.load.home=false -N -Qc aaa /home/runner/work/rizin/rizin/test/bins/fuzzed/poc-radare2-invalid-memory-access-elf-758
-- stdout

-- stderr
[ ] Analyze all flags starting with sym. and entry0 (aa)
[
[x] Analyze all flags starting with sym. and entry0 (aa)

[ ] Analyze function calls
[
[x] Analyze function calls
[ ] find and analyze function preludes
[
-- exit status: -1
[**]     /home/runner/work/rizin/rizin/test/bins/fuzzed      592 OK         0 BR        1 XX        0 FX
```

**Pull request merged**